3-platform/shopping-cart/db/postgres/01-create-shopping-cart.sql

```diff
@@ -0,0 +1,16 @@
+-- DROPPING TABLE IF EXISTS
+DROP TABLE IF EXISTS shopping_cart;
+
+-- CREATING TABLE "Shopping cart"
+CREATE TABLE shopping_cart (
+                               cart_id VARCHAR(255) NOT NULL,
+                               product_ids TEXT[],
+                               PRIMARY KEY (cart_id)
```

Comment:
There should not be PK. Create only index on cart_id.

Reply:
I agree, fixed it.